Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -2372,6 +2375,8 @@ + (void)forceContentUpdate:(LeanplumVariablesChangedBlock)block
         NSDictionary *regions = response[LP_KEY_REGIONS];
         NSDictionary *variantDebugInfo = [self parseVariantDebugInfoFromResponse:response];
         [[LPVarCache sharedCache] setVariantDebugInfo:variantDebugInfo];
+        NSDictionary *filenameToURL = [self parseFileURLsFromResponse:response];
```

Comment:
This is a list of files. Can we please rename this to something like "filenameToUrls"

Reply:
Also, 1 check , if "filenameToUrls" is nil, then not to assign it to [LPFileTransferManager sharedInstance].filenameToURL

Reply:
We don't want that check. If for whatever reason, we decide to stop sending the dictionary from the backend, we don't want the last non-nil value to be retained forever.